src/devices/bus/msx/slot/music.cpp

```diff
@@ -9,27 +9,14 @@ DEFINE_DEVICE_TYPE(MSX_SLOT_MUSIC, msx_slot_music_device, "msx_slot_music", "MSX
 
 msx_slot_music_device::msx_slot_music_device(const machine_config &mconfig, const char *tag, device_t *owner, u32 clock)
 	: msx_slot_rom_device(mconfig, MSX_SLOT_MUSIC, tag, owner, clock)
-	, m_ym2413(nullptr)
-	, m_ym2413_tag(nullptr)
+	, m_ym2413(*this, "^ym2413")
```

Comment:
Please initialise this with `m_ym2413(*this, finder_base::DUMMY_TAG)` and let the user configure it.  Making assumptions about the host system never ends well – we’re trying to eliminate devices that construct object finders with root or parent references in tags.